tdp/core/variables/cluster_variables.py

```diff
@@ -97,9 +97,14 @@ def initialize_cluster_variables(
                     services_initialized_by_this_function.add(service)
 
                 if service in services_initialized_by_this_function:
-                    service_variables.update_from_variables_folder(
-                        "add variables from " + collection_name, path
-                    )
+                    try:
+                        service_variables.update_from_variables_folder(
+                            "add variables from " + collection_name, path
+                        )
+                    except EmptyCommit:
+                        logger.warning(
```

Comment:
Why put a logger here ? Why `warning` ? Why you do not raise the exception to the caller ? I do not understand the logic of this PR...

Reply:
> Why put a logger here ? [...] Why you do not raise the exception to the caller ?

Because in my understanding it's a non blocking exception.

Furthermore, I took inspiration from the rest of the method: https://github.com/TOSIT-IO/tdp-lib/blob/97add22a3ca7bf4185a72d17f20ff24f250ba60f/tdp/core/variables/cluster_variables.py#L66-L107

- > Why warning ?

Because that's what was suggested in #324. I could put a log if needed.

Reply:
The problem is that you interpret an `except EmptyCommit` saying that the override file is empty but that is not what the `EmptyCommit` means. `EmptyCommit` means that there is no change following the merge of variables and this can happen if the override file is filled with variables identical to what is in the repository currently.

Reply:
This is true and the Warning log mesage is not appropriate but the issue still remains. The `EmptyCommit`  exception during validation should not interrupt the `init`

Reply:
Yes the issue should be resolved but this changes should not be merged, this is not the correct way to fix it.

Reply:
Ok I understand now. Well, we may still want to keep this except in the case where the override file doesn't modify anything?

Reply:
As discussed, I will rephrase the message in a new commit!

Reply:
Done, I'll squash at merge.